src/NavigationDots/NavigationDots.style.ts

```diff
@@ -4,18 +4,19 @@ import { transition } from '../animations'
 import { theme } from '../theme'
 
 export const NavigationDotsContainer = styled.div<{ width: number }>`
-  align-self: center;
   display: flex;
+  align-items: center;
 
   width: ${({ width }) => width}px;
 `
 
 export const Dot = styled.div<{ secondary?: boolean }>`
```

Comment:
What is `secondary` used for?

Reply:
I am not sure, but it has [its own ticket ](https://habxfr.atlassian.net/browse/APP-6157)2 years ago 😬 But I am not sure it is used ... I think I can safely remove it